### DISCUSSION TITLE
Use existing MariaDB with speedtest-tracker

### DISCUSSION THREAD

**Discussion by mahen57** [alexjustesen/speedtest-tracker#1646] [Q&A]
I want to run speedtest-tracker on my QNAP and have an existing MariaDB 10 running on the NAS.  Is there a documented way to spin up speedtest-tracker and point it to an existing MariaDB database to use?

**Comment by alexjustesen** [comment]
Just point your install at your database, you can use the MySQL/MariaDB instructions here: https://docs.speedtest-tracker.dev/getting-started/installation/installation
